### PR TITLE
Room enhancement

### PR DESCRIPTION
**Pull request by skystar-p**
Changes:
- Provide `player_count` in room list
- If room cache is not exist, delete room and raise error
- `./manage.py cleanroom` will clean all `Room` models

Todo:
- [x] Write tests

### CODE REVIEW

File: backend/api/management/commands/cleanroom.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+from api.models import Room
+
+
+class Command(BaseCommand):
+    help = 'command for cleaning Room model'
```
Comment (tirr-c) [inline backend/api/management/commands/cleanroom.py]: :+1:

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit bb795f3 has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit bb795f3c97e6c56f694954206dff8ad8c4afde3a with merge 90167b49c9912067f57f3650432f60c1878197fe...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/317006698?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing 90167b49c9912067f57f3650432f60c1878197fe to master...

**Pull request merged**